packages/create-bison-app/template/README.md.ejs

```diff
@@ -60,97 +61,212 @@ Bison uses [Nexus Schema](https://nexusjs.org/docs/) to create the GraphQL API.
 - [ ] Create a new GraphQL module using `yarn g:graphql organization`
 - [ ] Edit the new module to reflect what you want to expose via the API. In the following Mutation example, we alias the Mutation name, require a user to be logged in, and force the new Organization to be owned by the logged in user. All in about 10 lines of code!
 
-Because Nexus is strongly typed, all of the `t.` operations should autocomplete in your editor. Bison uses the [Prisma plugin](https://nexusjs.org/docs/pluginss/prisma/api) for Nexus, which enables the `t.model` and `t.crud` functions.
+Because Nexus is strongly typed, all of the `t.` operations should autocomplete in your editor.
+
+<details>
+  <summary>File: ./graphql/modules/organization.ts</summary>
+
+  ```ts
+  import { objectType, inputObjectType, queryField, mutationField, arg, list, nonNull } from 'nexus';
+
+  import { isAdmin } from '../../services/permissions';
+
+  // Organization Type
+  export const Organization = objectType({
+    name: 'Organization',
+    description: 'A Organization',
+    definition(t) {
+      t.nonNull.id('id');
+      t.nonNull.date('createdAt');
+      t.nonNull.date('updatedAt');
+      t.nonNull.string('name');
+
+      t.nonNull.list.nonNull.field('users', {
+        type: 'User',
+        resolve: async (parent, _, context) => {
+          return context.prisma.organization
+            .findUnique({
+              where: { id: parent.id },
+            })
+            .users();
+        },
+      });
+    },
+  });
 
-```ts
-// Organization Type
-export const Organization = objectType({
-  name: 'Organization',
-  description: 'A Organization',
-  definition(t) {
-    t.model.id();
-    t.model.name();
-    t.model.createdAt();
-    t.model.updatedAt();
-  },
-});
+  // Queries
+  export const findOrganizationsQuery = queryField('organizations', {
+    type: list('Organization'),
+    authorize: (_root, _args, ctx) => !!ctx.user,
```

Comment:
I am leaning towards leaving it out of the readme because it already has so much information in it. Maybe that belongs in a separate coding conventions or style guide?